mailbox/api/src/main/java/org/apache/james/mailbox/events/MailboxEvents.java

```diff
@@ -536,6 +536,25 @@ public final int hashCode() {
         }
     }
 
+    record MessageContentDeletionEvent(EventId eventId, Username username, MailboxId mailboxId, MessageId messageId, long size,
+                                       Instant internalDate, boolean hasAttachments, String headerBlobId, String bodyBlobId) implements Event {
```

Comment:
```suggestion
                                       Instant internalDate, boolean hasAttachments, BlobId headerBlobId, BlobId bodyBlobId) implements Event {
```

?

Reply:
Is it ok to add the `blob-api` dependency in the `mailbox-api` module?